<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FG_ChanMapDialog</class>
 <widget class="QDialog" name="FG_ChanMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Framegrabber Channel Mapping</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Acq Mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="acqModeLbl">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>35</red>
              <green>86</green>
              <blue>3</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>35</red>
              <green>86</green>
              <blue>3</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>MODE NAME</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string># Chans:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nChansLbl">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>74</red>
              <green>34</green>
              <blue>77</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>74</red>
              <green>34</green>
              <blue>77</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2304</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Specify a tab or space-delimited, zero (or 1) indexed list of channels for the electrode mapping. The mapping should look like a table, with regular rows and columns.  Rows terminate with a newline. The layout of the table will determine the initial layout of the &quot;Spatial Visualization&quot; window. The numbers in each cell of the table determine which channel (16bit pixel) from the framegrabber frame gets mapped to that cell's position.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="chanMapTE">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="plainText">
      <string>1	65	129	193	257	321	385	449	513	577	641	705	769	833	897	961	1025	1089	1153	1217	1281	1345	1409	1473	1537	1601	1665	1729	1793	1857	1921	1985	2049	2113	2177	2241
2	66	130	194	258	322	386	450	514	578	642	706	770	834	898	962	1026	1090	1154	1218	1282	1346	1410	1474	1538	1602	1666	1730	1794	1858	1922	1986	2050	2114	2178	2242
3	67	131	195	259	323	387	451	515	579	643	707	771	835	899	963	1027	1091	1155	1219	1283	1347	1411	1475	1539	1603	1667	1731	1795	1859	1923	1987	2051	2115	2179	2243
4	68	132	196	260	324	388	452	516	580	644	708	772	836	900	964	1028	1092	1156	1220	1284	1348	1412	1476	1540	1604	1668	1732	1796	1860	1924	1988	2052	2116	2180	2244
5	69	133	197	261	325	389	453	517	581	645	709	773	837	901	965	1029	1093	1157	1221	1285	1349	1413	1477	1541	1605	1669	1733	1797	1861	1925	1989	2053	2117	2181	2245
6	70	134	198	262	326	390	454	518	582	646	710	774	838	902	966	1030	1094	1158	1222	1286	1350	1414	1478	1542	1606	1670	1734	1798	1862	1926	1990	2054	2118	2182	2246
7	71	135	199	263	327	391	455	519	583	647	711	775	839	903	967	1031	1095	1159	1223	1287	1351	1415	1479	1543	1607	1671	1735	1799	1863	1927	1991	2055	2119	2183	2247
8	72	136	200	264	328	392	456	520	584	648	712	776	840	904	968	1032	1096	1160	1224	1288	1352	1416	1480	1544	1608	1672	1736	1800	1864	1928	1992	2056	2120	2184	2248
9	73	137	201	265	329	393	457	521	585	649	713	777	841	905	969	1033	1097	1161	1225	1289	1353	1417	1481	1545	1609	1673	1737	1801	1865	1929	1993	2057	2121	2185	2249
10	74	138	202	266	330	394	458	522	586	650	714	778	842	906	970	1034	1098	1162	1226	1290	1354	1418	1482	1546	1610	1674	1738	1802	1866	1930	1994	2058	2122	2186	2250
11	75	139	203	267	331	395	459	523	587	651	715	779	843	907	971	1035	1099	1163	1227	1291	1355	1419	1483	1547	1611	1675	1739	1803	1867	1931	1995	2059	2123	2187	2251
12	76	140	204	268	332	396	460	524	588	652	716	780	844	908	972	1036	1100	1164	1228	1292	1356	1420	1484	1548	1612	1676	1740	1804	1868	1932	1996	2060	2124	2188	2252
13	77	141	205	269	333	397	461	525	589	653	717	781	845	909	973	1037	1101	1165	1229	1293	1357	1421	1485	1549	1613	1677	1741	1805	1869	1933	1997	2061	2125	2189	2253
14	78	142	206	270	334	398	462	526	590	654	718	782	846	910	974	1038	1102	1166	1230	1294	1358	1422	1486	1550	1614	1678	1742	1806	1870	1934	1998	2062	2126	2190	2254
15	79	143	207	271	335	399	463	527	591	655	719	783	847	911	975	1039	1103	1167	1231	1295	1359	1423	1487	1551	1615	1679	1743	1807	1871	1935	1999	2063	2127	2191	2255
16	80	144	208	272	336	400	464	528	592	656	720	784	848	912	976	1040	1104	1168	1232	1296	1360	1424	1488	1552	1616	1680	1744	1808	1872	1936	2000	2064	2128	2192	2256
17	81	145	209	273	337	401	465	529	593	657	721	785	849	913	977	1041	1105	1169	1233	1297	1361	1425	1489	1553	1617	1681	1745	1809	1873	1937	2001	2065	2129	2193	2257
18	82	146	210	274	338	402	466	530	594	658	722	786	850	914	978	1042	1106	1170	1234	1298	1362	1426	1490	1554	1618	1682	1746	1810	1874	1938	2002	2066	2130	2194	2258
19	83	147	211	275	339	403	467	531	595	659	723	787	851	915	979	1043	1107	1171	1235	1299	1363	1427	1491	1555	1619	1683	1747	1811	1875	1939	2003	2067	2131	2195	2259
20	84	148	212	276	340	404	468	532	596	660	724	788	852	916	980	1044	1108	1172	1236	1300	1364	1428	1492	1556	1620	1684	1748	1812	1876	1940	2004	2068	2132	2196	2260
21	85	149	213	277	341	405	469	533	597	661	725	789	853	917	981	1045	1109	1173	1237	1301	1365	1429	1493	1557	1621	1685	1749	1813	1877	1941	2005	2069	2133	2197	2261
22	86	150	214	278	342	406	470	534	598	662	726	790	854	918	982	1046	1110	1174	1238	1302	1366	1430	1494	1558	1622	1686	1750	1814	1878	1942	2006	2070	2134	2198	2262
23	87	151	215	279	343	407	471	535	599	663	727	791	855	919	983	1047	1111	1175	1239	1303	1367	1431	1495	1559	1623	1687	1751	1815	1879	1943	2007	2071	2135	2199	2263
24	88	152	216	280	344	408	472	536	600	664	728	792	856	920	984	1048	1112	1176	1240	1304	1368	1432	1496	1560	1624	1688	1752	1816	1880	1944	2008	2072	2136	2200	2264
25	89	153	217	281	345	409	473	537	601	665	729	793	857	921	985	1049	1113	1177	1241	1305	1369	1433	1497	1561	1625	1689	1753	1817	1881	1945	2009	2073	2137	2201	2265
26	90	154	218	282	346	410	474	538	602	666	730	794	858	922	986	1050	1114	1178	1242	1306	1370	1434	1498	1562	1626	1690	1754	1818	1882	1946	2010	2074	2138	2202	2266
27	91	155	219	283	347	411	475	539	603	667	731	795	859	923	987	1051	1115	1179	1243	1307	1371	1435	1499	1563	1627	1691	1755	1819	1883	1947	2011	2075	2139	2203	2267
28	92	156	220	284	348	412	476	540	604	668	732	796	860	924	988	1052	1116	1180	1244	1308	1372	1436	1500	1564	1628	1692	1756	1820	1884	1948	2012	2076	2140	2204	2268
29	93	157	221	285	349	413	477	541	605	669	733	797	861	925	989	1053	1117	1181	1245	1309	1373	1437	1501	1565	1629	1693	1757	1821	1885	1949	2013	2077	2141	2205	2269
30	94	158	222	286	350	414	478	542	606	670	734	798	862	926	990	1054	1118	1182	1246	1310	1374	1438	1502	1566	1630	1694	1758	1822	1886	1950	2014	2078	2142	2206	2270
31	95	159	223	287	351	415	479	543	607	671	735	799	863	927	991	1055	1119	1183	1247	1311	1375	1439	1503	1567	1631	1695	1759	1823	1887	1951	2015	2079	2143	2207	2271
32	96	160	224	288	352	416	480	544	608	672	736	800	864	928	992	1056	1120	1184	1248	1312	1376	1440	1504	1568	1632	1696	1760	1824	1888	1952	2016	2080	2144	2208	2272
33	97	161	225	289	353	417	481	545	609	673	737	801	865	929	993	1057	1121	1185	1249	1313	1377	1441	1505	1569	1633	1697	1761	1825	1889	1953	2017	2081	2145	2209	2273
34	98	162	226	290	354	418	482	546	610	674	738	802	866	930	994	1058	1122	1186	1250	1314	1378	1442	1506	1570	1634	1698	1762	1826	1890	1954	2018	2082	2146	2210	2274
35	99	163	227	291	355	419	483	547	611	675	739	803	867	931	995	1059	1123	1187	1251	1315	1379	1443	1507	1571	1635	1699	1763	1827	1891	1955	2019	2083	2147	2211	2275
36	100	164	228	292	356	420	484	548	612	676	740	804	868	932	996	1060	1124	1188	1252	1316	1380	1444	1508	1572	1636	1700	1764	1828	1892	1956	2020	2084	2148	2212	2276
37	101	165	229	293	357	421	485	549	613	677	741	805	869	933	997	1061	1125	1189	1253	1317	1381	1445	1509	1573	1637	1701	1765	1829	1893	1957	2021	2085	2149	2213	2277
38	102	166	230	294	358	422	486	550	614	678	742	806	870	934	998	1062	1126	1190	1254	1318	1382	1446	1510	1574	1638	1702	1766	1830	1894	1958	2022	2086	2150	2214	2278
39	103	167	231	295	359	423	487	551	615	679	743	807	871	935	999	1063	1127	1191	1255	1319	1383	1447	1511	1575	1639	1703	1767	1831	1895	1959	2023	2087	2151	2215	2279
40	104	168	232	296	360	424	488	552	616	680	744	808	872	936	1000	1064	1128	1192	1256	1320	1384	1448	1512	1576	1640	1704	1768	1832	1896	1960	2024	2088	2152	2216	2280
41	105	169	233	297	361	425	489	553	617	681	745	809	873	937	1001	1065	1129	1193	1257	1321	1385	1449	1513	1577	1641	1705	1769	1833	1897	1961	2025	2089	2153	2217	2281
42	106	170	234	298	362	426	490	554	618	682	746	810	874	938	1002	1066	1130	1194	1258	1322	1386	1450	1514	1578	1642	1706	1770	1834	1898	1962	2026	2090	2154	2218	2282
43	107	171	235	299	363	427	491	555	619	683	747	811	875	939	1003	1067	1131	1195	1259	1323	1387	1451	1515	1579	1643	1707	1771	1835	1899	1963	2027	2091	2155	2219	2283
44	108	172	236	300	364	428	492	556	620	684	748	812	876	940	1004	1068	1132	1196	1260	1324	1388	1452	1516	1580	1644	1708	1772	1836	1900	1964	2028	2092	2156	2220	2284
45	109	173	237	301	365	429	493	557	621	685	749	813	877	941	1005	1069	1133	1197	1261	1325	1389	1453	1517	1581	1645	1709	1773	1837	1901	1965	2029	2093	2157	2221	2285
46	110	174	238	302	366	430	494	558	622	686	750	814	878	942	1006	1070	1134	1198	1262	1326	1390	1454	1518	1582	1646	1710	1774	1838	1902	1966	2030	2094	2158	2222	2286
47	111	175	239	303	367	431	495	559	623	687	751	815	879	943	1007	1071	1135	1199	1263	1327	1391	1455	1519	1583	1647	1711	1775	1839	1903	1967	2031	2095	2159	2223	2287
48	112	176	240	304	368	432	496	560	624	688	752	816	880	944	1008	1072	1136	1200	1264	1328	1392	1456	1520	1584	1648	1712	1776	1840	1904	1968	2032	2096	2160	2224	2288
49	113	177	241	305	369	433	497	561	625	689	753	817	881	945	1009	1073	1137	1201	1265	1329	1393	1457	1521	1585	1649	1713	1777	1841	1905	1969	2033	2097	2161	2225	2289
50	114	178	242	306	370	434	498	562	626	690	754	818	882	946	1010	1074	1138	1202	1266	1330	1394	1458	1522	1586	1650	1714	1778	1842	1906	1970	2034	2098	2162	2226	2290
51	115	179	243	307	371	435	499	563	627	691	755	819	883	947	1011	1075	1139	1203	1267	1331	1395	1459	1523	1587	1651	1715	1779	1843	1907	1971	2035	2099	2163	2227	2291
52	116	180	244	308	372	436	500	564	628	692	756	820	884	948	1012	1076	1140	1204	1268	1332	1396	1460	1524	1588	1652	1716	1780	1844	1908	1972	2036	2100	2164	2228	2292
53	117	181	245	309	373	437	501	565	629	693	757	821	885	949	1013	1077	1141	1205	1269	1333	1397	1461	1525	1589	1653	1717	1781	1845	1909	1973	2037	2101	2165	2229	2293
54	118	182	246	310	374	438	502	566	630	694	758	822	886	950	1014	1078	1142	1206	1270	1334	1398	1462	1526	1590	1654	1718	1782	1846	1910	1974	2038	2102	2166	2230	2294
55	119	183	247	311	375	439	503	567	631	695	759	823	887	951	1015	1079	1143	1207	1271	1335	1399	1463	1527	1591	1655	1719	1783	1847	1911	1975	2039	2103	2167	2231	2295
56	120	184	248	312	376	440	504	568	632	696	760	824	888	952	1016	1080	1144	1208	1272	1336	1400	1464	1528	1592	1656	1720	1784	1848	1912	1976	2040	2104	2168	2232	2296
57	121	185	249	313	377	441	505	569	633	697	761	825	889	953	1017	1081	1145	1209	1273	1337	1401	1465	1529	1593	1657	1721	1785	1849	1913	1977	2041	2105	2169	2233	2297
58	122	186	250	314	378	442	506	570	634	698	762	826	890	954	1018	1082	1146	1210	1274	1338	1402	1466	1530	1594	1658	1722	1786	1850	1914	1978	2042	2106	2170	2234	2298
59	123	187	251	315	379	443	507	571	635	699	763	827	891	955	1019	1083	1147	1211	1275	1339	1403	1467	1531	1595	1659	1723	1787	1851	1915	1979	2043	2107	2171	2235	2299
60	124	188	252	316	380	444	508	572	636	700	764	828	892	956	1020	1084	1148	1212	1276	1340	1404	1468	1532	1596	1660	1724	1788	1852	1916	1980	2044	2108	2172	2236	2300
61	125	189	253	317	381	445	509	573	637	701	765	829	893	957	1021	1085	1149	1213	1277	1341	1405	1469	1533	1597	1661	1725	1789	1853	1917	1981	2045	2109	2173	2237	2301
62	126	190	254	318	382	446	510	574	638	702	766	830	894	958	1022	1086	1150	1214	1278	1342	1406	1470	1534	1598	1662	1726	1790	1854	1918	1982	2046	2110	2174	2238	2302
63	127	191	255	319	383	447	511	575	639	703	767	831	895	959	1023	1087	1151	1215	1279	1343	1407	1471	1535	1599	1663	1727	1791	1855	1919	1983	2047	2111	2175	2239	2303
64	128	192	256	320	384	448	512	576	640	704	768	832	896	960	1024	1088	1152	1216	1280	1344	1408	1472	1536	1600	1664	1728	1792	1856	1920	1984	2048	2112	2176	2240	2304
																																			
</string>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FG_ChanMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FG_ChanMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
